src/Bridges/bridge_optimizer.jl

```diff
@@ -1230,6 +1230,23 @@ function MOI.set(
     end
 end
 
+function MOI.supports(
+    ::AbstractBridgeOptimizer,
+    ::MOI.ConstraintName,
+    ::Type{MOI.ConstraintIndex{MOI.SingleVariable,S}},
+) where {S}
+    return false
```

Comment:
Shouldn't we throw the error here as well, I'm hesitant

Reply:
Why would we throw an error on `supports`?

Reply:
This is not a valid call, it's like calling `supports` with a non-copyable attribute which is throwing an error.
Otherwise, the user would think that it's an issue with the model not supporting it but in fact, it's not valid to set a constraint name for such constraint.

Reply:
How's this now.